src/bootstrap/native.rs

```diff
@@ -446,6 +446,7 @@ impl Step for Openssl {
             "powerpc64-unknown-linux-gnu" => "linux-ppc64",
             "powerpc64le-unknown-linux-gnu" => "linux-ppc64le",
             "s390x-unknown-linux-gnu" => "linux64-s390x",
+            "sparc64-unknown-linux-gnu" => "linux-sparc64",
```

Comment:
I believe it's actually `linux64-sparcv9`: https://github.com/libtor/openssl/blob/1ad38e5/Configure#L385

Reply:
Oh, good catch, thank you. I was actually unsure about this but my tests aren't finished yet.

Will get back in a few minutes.

Reply:
I have changed that now. I'm now testing whether we need to pass "-m64" as well (see the case for BSD-sparc64 further below).

Reply:
No, OpenSSL puts -m64 in there already for linux64-sparcv9.